### PR TITLE
Update to new release 3.6. Update dependency of joblib.

### PR DESCRIPTION
**Pull request by twiecki**
Closes https://github.com/conda-forge/pymc3-feedstock/issues/32

### CODE REVIEW

**Comment by ocefpaf** [comment]
> Duplicate of #34? I think we don't have to worry about updating on our own, a bot will do it for us .

Humans are still first class citizens here, so I closed #34 in favor of this one ;-)

File: recipe/meta.yaml [hunk 1]
```diff
@@ -56,3 +56,4 @@ extra:
   recipe-maintainers:
     - ericmjl
     - springcoil
+    - twiecki
```
Comment (ocefpaf) [inline recipe/meta.yaml]: :+1:

File: recipe/meta.yaml [hunk 2]
```diff
@@ -21,10 +21,10 @@ requirements:
     - numpy
     - scipy >=0.12.0
     - matplotlib >=1.5.0
-    - theano >=0.9
+    - theano >=1.0.0
     - pandas >=0.18.0
     - patsy >=0.4.0
-    - joblib >=0.9
+    - joblib <0.13.0
```
Comment (ocefpaf) [inline recipe/meta.yaml]: BTW, the bot does not updated the dependencies yet, so this is always welcome!

**Comment by twiecki** [comment]
@ocefpaf I appreciate that, although it seems that https://github.com/conda-forge/pymc3-feedstock/pull/34/files made some other meaningful changes. Should we port those over somehow?

Also seems like the linux build is broken? 
image

**Comment by ocefpaf** [comment]
It is only the re-rendering. We can do that with the webservice. Let me just check the Linux failure first.

**Pull request merged**